cmds/core/uptime/uptime.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Get the time the machine has been up
+//
+// Synopsis:
+//     uptime
+//
+
```

Comment:
Remove this empty line. The package comment must be immediately before "package main"

Reply:
Ping on this?